### DISCUSSION TITLE
Mikrotik router to PHPNEXBILL

### DISCUSSION THREAD

**Discussion by patricknana532** [hotspotbilling/phpnuxbill#343] [General]
Please need help I want to add my mikrotik router to PHPNEXBILL that I already host in my hosting. how do I do it please?

**Comment by gachezra** [comment]
As of now I'm still wondering how i can do that, if you ever found a way please let me know.

**Comment by ibnux** [comment]
your router need Public IP and can be accessed froum your hosting, your hosting must open port, as most hosting disable port access.

if your mikrotik has public IP, then you need to allow access port API from Internet, this need some firewall configuration

**Reply by gachezra** [reply]
Say I host the PHPNUXBILL software locally, will it connect to the local IP address given to the router?

**Reply by bangprans** [reply]
pak apakah bisa menggunakan alamat VPN remote pihak ke-3? contoh tunnel-web (remote mikrotik)
